hack/update/golang_version/update_golang_version.go

```diff
@@ -185,6 +185,11 @@ var (
 				`GO_VERSION: .*`: `GO_VERSION: '{{.StableVersion}}'`,
 			},
 		},
+		".github/workflows/inspektor-gadget-version.yml": {
```

Comment:
now that we merged that other Pr that updates all go-versions this is probably redundant ?

Reply:
Yup, I'll rebase this now